app/views/admin/booths/_change_state_dropdown.html.haml

```diff
@@ -1,17 +1,20 @@
 - if booth.transition_possible? :accept
   %li= link_to 'Accept booth',
   accept_admin_conference_booth_path(@conference.short_title, booth),
-  method: :patch, id: "accept_booth_#{booth.id}"
+  method: :patch ,id: "accept_booth_#{booth.id}",
+  data: { confirm: 'Are you sure? By accepting this booth request an email is going to be sent informing the submitter about the acceptance. If you are not completely sure yet, you can change the state to \'To accept\'.' }
```

Comment:
What if the organizers do not read this text? How can they know an email is going to be sent out?